sale_operating_unit/models/sale_order.py

```diff
@@ -77,3 +90,12 @@ class SaleOrderLine(models.Model):
         string="Operating Unit",
         store=True,
     )
+
+    def _prepare_invoice_line(self, **optional_values):
+        values = super()._prepare_invoice_line(**optional_values)
+        values.update(
+            {
+                "operating_unit_id": self.operating_unit_id.id,
```

Comment:
you don't need to set "operating_unit_id" for account.move.line. it will be handle by: https://github.com/OCA/operating-unit/blob/95d72a43f22c1c6c87228cd4b4339268026a16ff/account_operating_unit/models/account_move.py#L18

![image](https://github.com/OCA/operating-unit/assets/8724793/2a063fa4-bb49-4da3-ad54-bae8a10e1648)